zschema/tests.py

```diff
@@ -149,7 +153,7 @@ def setUp(self):
         })
         self.host = Record({
                 "ipstr":IPv4Address(required=True),
-                "ip":Long(),
+                "ip":Unsigned32BitInteger(),
```

Comment:
I assume this change (and the similar change `example.py`) is because `Long` is deprecated?